packages/react-graphql/src/async/query.ts

```diff
@@ -22,7 +22,9 @@ export function createAsyncQuery<Data, Variables, DeepPartial>({
     return useAsync(resolver, {assets: AssetTiming.NextPage}).load;
   }
 
-  function usePrefetch(options: VariableOptions<Variables>) {
+  function usePrefetch(
+    options: VariableOptions<Variables> & Pick<QueryProps, 'fetchPolicy'>,
```

Comment:
wasn't sure whether it would be better to do `options: VariableOptions<Variables> & Omit<WatchQueryOptions, 'query'>` instead.

Reply:
I think it's fine to start here, everyone was fine with no options for a long time, and I bet they will be happy with just this one extra option for a long time from now :)

Reply:
Ok great. Thanks Chris!